share/src/main/java/io/zensoft/share/service/diesel/body/LoginPostRequestBody.java

```diff
@@ -0,0 +1,30 @@
+package io.zensoft.share.service.diesel.body;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+@Data
+@Component
+public class LoginPostRequestBody {
+    private final String fauth_key = "880ea6a14ea49e853634fbdc5015a024";
+    private final String freferer = "http://diesel.elcat.kg/";
+
+    @Value( "${diesel.password}" )
+    private String ips_password;
+    @Value( "${diesel.username}" )
+    private String ips_username;
```

Comment:
It injects from appication.properties file?

Reply:
No, file where I take values I didn't commited)

Reply:
Ok. I am not good enough at Spring but i think Spring won't find your properties file, it will search from application.properties

Reply:
Ok, I will show it directly.